src/common/options/mds.yaml.in

```diff
@@ -5,7 +5,7 @@ options:
 - name: mds_alternate_name_max
   type: size
   level: advanced
-  desc: set the maximum length of alternate names for dentries
+  desc: Set the maximum length of alternate names for dentries
```

Comment:
I don't care much about changing this but I thought the existing convention was to begin with a lowercase. If you want to change it, I support that but it should be done everywhere.

Reply:
I found a mix of existing entries, I've been harmonizing to capitalizing elsewhere.  @zdover23 do you have thoughts on which way this should be?

Reply:
I'm pushing the below rewording as well.  I had the sense that it was a watchdog timeout of sorts.